EIPS/eip-7557.md

```diff
@@ -0,0 +1,403 @@
+---
+eip: 7557
+title: Block-level Warming with fair cost savings
+description: Block-level warming of addresses and slots with access lists
+author: Yoav Weiss (@yoavw), Alex Forshtat (@forshtat), Dror Tirosh (@drortirosh), Shahaf Nacson (@shahafn)
+discussions-to: https://ethereum-magicians.org/t/eip-7557-block-level-warming/16642
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-10-01
+---
+
+## Abstract
+
+A mechanism for a fair distribution of the gas costs associated with access to addresses and storage slots
+among multiple transactions with shared items in their `accessList`.
```

Comment:
The abstract should contain a brief (but still technical) overview of the proposal. Can you sketch out the mechanism here without getting too in the weeds?